README.md

```diff
@@ -1,37 +1,216 @@
+### Expiry
 
 Delete submissions and their results when their TTL expires.
 
-##### Ingester
+```bash
+docker run --name expiry cccs/assemblyline-core python -m assemblyline_core.expiry.run_expiry
+```
+
+### Ingester
 
 Move ingested files from the priority queues to the processing queues.
 
-##### Metrics
+```bash
+docker run --name ingester cccs/assemblyline-core python -m assemblyline_core.ingester
+```
+
+### Metrics
 
 Generates metrics of the different components in the system.
 
-##### Scaler
+#### Heartbeat Manager
+
+```bash
+docker run --name heartbeat cccs/assemblyline-core python -m assemblyline_core.metrics.run_heartbeat_manager
+```
+
+#### Metrics Aggregator
+
+```bash
+docker run --name metrics cccs/assemblyline-core python -m assemblyline_core.metrics.run_metrics_aggregator
+```
+
+#### Statistics Aggregator
+
+```bash
+docker run --name statistics cccs/assemblyline-core python -m assemblyline_core.metrics.run_statistics_aggregator
+```
+
+### Scaler
 
 Spin up and down services in the system depending on the load.
 
-##### Updater
+```bash
+docker run --name scaler cccs/assemblyline-core python -m assemblyline_core.scaler.run_scaler
+```
+
+### Updater
 
 Make sure the different services get their latest update files.
 
-##### Workflow
+```bash
+docker run --name updater cccs/assemblyline-core python -m assemblyline_core.updater.run_updater
+```
+
+### Workflow
 
 Run the different workflows in the system and apply their labels, priority and status.
+
+```bash
+docker run --name workflow cccs/assemblyline-core python -m assemblyline_core.workflow.run_workflow
+```
+
+## Documentation
+
+For more information about these Assemblyline components, follow this [overview](https://cybercentrecanada.github.io/assemblyline4_docs/overview/architecture/) of the system's architecture.
+
+---
+
+# Assemblyline 4 - Core
+
+Ce dépôt fournit des services de base pour Assemblyline 4.
+
+## Variantes d'images et balises
+
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
+|      build_type      | Le type de compilation utilisé. `dev` est la dernière version instable. `stable` est la dernière version stable. `stable` ou `dev` |     `stable` ou `dev`      |
+|        series        | Le type de build utilisé. `dev` est le dernier build unstable : `version.buildType`.                                               | `4.5.stable`, `4.5.1.dev3` |
+
+## Composants
+
+### Alerter
+
+Créer des alertes pour les différentes soumissions dans le système.
```

Comment:
First of all the tense used for all the components should be consistent for visual. So it should be : "Crée" instead of "Créer" or else change all the others to this tense.